README.md

```diff
@@ -58,38 +83,28 @@ smtphelo = example.org
 [tagmap]
 all: me@example.com
 webserver, !mailserver: httpadmins@example.com, you@example.com
-~~~
+```
 
-Instead of specifying `smtpserver`, `smtpport` and `smtphelo`, you can specify the `sendmail` option with a path to your sendmail binary (defaulted to `/usr/sbin/sendmail`). If you do not specify `smtpserver`, tagmail will default to using sendmail.
+If you specify `sendmail` instead of `smtpserver`, it might look like:
 
-~~~
+```
 [transport]
 reportfrom = reports@example.org
 sendmail = /usr/sbin/sendmail
 
 [tagmap]
 all: me@example.com
 webserver, !mailserver: httpadmins@example.com, you@example.com
-~~~
-
-Each line in the `[tagmap]` section of `tagmail.conf` should include a comma-separated list of tags, a colon, and a comma-separated list of email addresses to receive log messages containing the provided tags.
-
-If you prefix a tag with an exclamation mark, Puppet subtracts any messages with that tag from the line's results.
-
-Puppet's [loglevels](https://docs.puppetlabs.com/references/latest/metaparameter.html#loglevel) (`debug`, `info`, `notice`, `warning`, `err`, `alert`, `emerg`, `crit`, and `verbose`) can also be used as tags, and the `all` tag always matches every log message.
-
-The above example sends all log messages to `me@example.com`, and all messages from webservers that are not also from mailservers to `httpadmins@example.com` and to `you@example.com`.
-
+```
 
 ## Limitations
 
-This module supports Puppet Enterprise and Puppet versions 3.8 or newer. For older versions of Puppet, use Puppet's built-in tagmail feature.
-
+This module should be used only with Puppet Enterprise and Puppet versions 3.7 or newer, and only if you're using the JVM on the Puppet master. For older versions of Puppet, or if using the legacy Puppet master on Apache/Rack/Passenger, use Puppet's built-in tagmail feature.
```

Comment:
In the requirements section we say Puppet 3.8 as the lower bound and i think that's what it should be here too.